tests/no_lattice.rs

```diff
@@ -107,6 +107,47 @@ pub async fn kvcounter_basic() -> Result<()> {
     Ok(())
 }
 
+pub async fn actor_to_actor_call_alias() -> Result<()> {
+    ::std::env::remove_var("KVCACHE_NATS_URL");
+    let h = HostBuilder::new().with_namespace("actor2actor").build();
+    h.start().await?;
+    let pinger = Actor::from_file("./tests/modules/pinger.wasm")?;
+    let ponger = Actor::from_file("./tests/modules/ponger.wasm")?;
```

Comment:
Subtle but cool, this host has no NATS but the actors can still call each other. Awesome.

Reply:
actors and capability providers can all talk to each other locally within a host. In fact, they even do that as a prioritized call when NATS _is_ running (e.g. if the target of an invocation is both locally inside a host and outside on the lattice, the local one is invoked).